Pcap++/header/PcapFilter.h

```diff
@@ -118,7 +120,7 @@ namespace pcpp
 		const std::string m_filterStr;
 
 	public:
-		BPFStringFilter(const std::string& filterStr) : m_filterStr(filterStr) {}
+		BPFStringFilter(const std::string& filterStr) : m_filterStr(filterStr) {}
```

Comment:
was there any change here?

Reply:
Nope. It's probably different line ending.